src/css/map-view.css

```diff
@@ -561,43 +562,68 @@ represents 1 unit of the given distance measurement. */
         background-color: var(--map-col-content-bkg, var(--map-col-bkg));
         color: var(--map-col-text-body, var(--map-col-text));
         font-family: var(--map-body-font);
-        border: 1px solid;
-        border-color: var(--map-col-border-muted);
+        border: none;
         box-shadow: unset;
 
         ::placeholder {
           color: var(--map-col-text-muted);
         }
-
-        &.search-input__error-input {
-          border-color: var(--map-col-input-error-border);
-          background-color: var(--map-col-input-error-bkg);
-        }
       }
     }
+
+    &:focus-within {
+      border-color: var(--map-col-border-highlight-search-box, var(--map-col-blue));
+    }
+
+    &.search-input__error-input,
+    &.search-input__error-input input[type=text].search-input__input {
+      border-color: var(--map-col-input-error-border);
+      background-color: var(--map-col-input-error-bkg);
+    }
   
-    .search-input__search-button, .search-input__cancel-button {
+    .search-input__cancel-button-container {
+      align-items: center;
+      display: flex;
+    }
+
+    .search-input__search-button,
+    .search-input__cancel-button {
       background-color: transparent;
-      border-color: transparent;
+      border: none;
       color: unset;
-      width: 24px;
-      height: 24px;
-      margin-right: 8px;
+      width: 32px;
+      height: 32px;
       display: inline-flex;
       justify-content: center;
       align-items: center;
       position: relative;
+      flex-shrink: 0;
+      margin: 4px;
   
       .search-input__button-icon {
         font-size: 16px;
       }
     }
 
+    .search-input__vertical-divider {
+      width: 0;
+      height:24px;
+      border-right: solid 2px var(--map-col-border-muted);
+    }
+
     .search-input__search-button {
       color: var(--map-col-buttons-icon-muted);
+      border-radius: 4px;
+
+      &.search-input__search-button--active {
+        color: var(--map-col-text-highlight, var(--map-col-text-body));
```

Comment:
--map-col-text-body might not work as a fallback since it's from the new set of theme variables. fwiw currentColor has worked for me when I need text colors

Reply:
instead of --map-col-text-highlight, can we use --map-col-buttons-icon?